src/bika/lims/content/analysisrequest.py

```diff
@@ -1645,26 +1646,61 @@ def getLate(self):
                 return True
         return False
 
+    def getRawReports(self, inlcude_provisional=True, **kw):
```

Comment:
We usually use the `getRaw*` function to return UIDs. I suggest to add the `full_objects=False` in `getReports` (along with **kwargs) instead.

Also note that `getRawReports` could rely on `get_backreferences`:

```python
def getRawReports(self):
    return get_backreferences(self, "ARReportAnalysisRequest")

def getReportUIDs(self):
    return self.getRawReports()

```

Also note there is a typo with `inlcude_provisional` (should be `include_provisional`)

```python
def getReports(self, include_provisional=True, full_objects=False, **kw):
    ...
```
```

Reply:
Yes, I thought about this. Shall I rely better on the backreferences instead?

Reply:
> Yes, I thought about this. Shall I rely better on the backreferences instead?

I think that relying on backreferences is better, yes